app/screens/post_options/components/options/follow_option.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo} from 'react';
+
+import {Screens} from '@constants';
+import {t} from '@i18n';
+
+import BaseOption from './base_option';
+
+type FollowThreadOptionProps = {
+    thread?: any;
+    location?: typeof Screens[keyof typeof Screens];
+};
+
+//todo: to implement CRT follow thread
+
+const FollowThreadOption = ({thread}: FollowThreadOptionProps) => {
+    //todo: to enable after CRT. Move this to PostOptions and do not mount the component
+    const config = useMemo(() => {
```

Comment:
Doesn't seem to need memo. The things passed down to the component are all base types (strings).

Reply:
Fixed.